.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
you should throw `.vscode/` in the `.gitignore`, or put it in your local global `.gitignore` (what i do)